src/components/Patient/ManagePatients.tsx

```diff
@@ -973,7 +976,7 @@ export const PatientManager = () => {
         <div className="mt-2 flex h-full flex-col gap-3 xl:flex-row">
           <div className="flex-1" id="total-patientcount">
             <CountBlock
-              text="Total Patients"
+              text="Total_Patients"
```

Comment:
wrap with `t` in the place where the key is being specified

```suggestion
              text={t("total_patients")}
```